src/components/ProductCard/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { StarRate, Wishlist, Basket } from "@shared";
+import { Link } from "react-router-dom";
+
+interface ProductProp {
+  img: string;
+  name: string;
+  type: string;
+  cost: number;
+  rate: number;
+}
+
+const ProductCardExample: ProductProp = {
```

Comment:
If u want to show example or ProducCard - just add explanation for each interface item. For example like this:

interface ProductProp {
  /** Source image link of product card, example: "https://.../image.jpg" */
  img: string;

  /** Translated name of product (or map of translations to get name dynamically based on selected language)*/
  name: string;

  /** Category of product (type of product) */
  type: string;
  ...
}

Reply:
When u will move interface to shared folder add these explanations for each field.